script/gutterball-debug.sh

```diff
@@ -0,0 +1,12 @@
+# error out if called directly
+if [ $BASH_SOURCE == $0 ]
+then
+  echo "This script should not be executed directly, use foreman-debug instead."
+  exit 1
+fi
+
+# Gutterball
+add_files "/var/log/gutterball/*"
+add_files "/etc/gutterball/certs/ampq/gutterball.truststore"
+add_files "/etc/candlepin/certs/ampq/candlepin.truststore"
+add_files "/etc/gutterball/gutterball.conf"
```

Comment:
I assume the truststore files do not contain sensitive information like private keys or passwords.

Foreman debug automatically detects things like `password=XYZ` and filters those out. If the conf file has different syntax, test it first and we can extend filtering a bit.